### PR TITLE
Fix 2 typos in state preparation tutorial

### PR DESCRIPTION
**Pull request by angusjlowe**
**Description of the Change:** Change purity to be less than or equal to one (not greater than) and fix spelling in word "parameters"

**Benefits:** n/a

**Possible Drawbacks:** n/a

**Related GitHub Issues:** n/a

### CODE REVIEW

**Comment by AroosaIjaz** [comment]
Ah I saw this too! Thanks Angus!

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,9 @@
 
 ### Documentation
 
+* fixed 2 typos in the state preparation tutorial
```
Comment (co9olguy) [inline .github/CHANGELOG.md]:
```suggestion
* Fixed typos in the state preparation tutorial
```

File: examples/pennylane_run_state_preparation.py [hunk 2]
```diff
@@ -71,7 +71,7 @@
 # number of layers in the circuit
 nr_layers = 2
 
-# randomly initialize parameteres from a normal distribution
+# randomly initialize parameters from a normal distribution
```
Comment (josh146) [inline examples/pennylane_run_state_preparation.py]: Good catch!

**Comment by co9olguy** [comment]
Thanks @angusjlowe. I approved, but there's one small bit of correction I'd like before merging. See suggestion on github (normally I'd just do myself, but the PR is coming from your personal repo)

File: .github/CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,9 @@
...
+[#321](https://github.com/XanaduAI/pennylane/pull/321)
```
Comment (josh146) [inline .github/CHANGELOG.md]: Note that in ReST formatting, bullet point continuations need to match the first indentation
```suggestion
   [#321](https://github.com/XanaduAI/pennylane/pull/321)
```

**Pull request merged**